### PR TITLE
Fix Portal Agent Banner partial cut issue, Optimize dashboard framework

### PR DESCRIPTION
**Pull request by DexterDreeeam**
**Linked GitHub issue ID**: #  

## Pull Request Checklist

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Code compiles correctly with all tests are passed.
- [x] I've read the [contributing guide](https://github.com/microsoft/HydraLab/blob/main/CONTRIBUTING.md#making-changes-to-the-code) and followed the recommended practices.
- [ ] [Wikis](https://github.com/microsoft/HydraLab/wiki) or [README](https://github.com/microsoft/HydraLab/blob/main/README.md) have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?
*If this introduces a breaking change for Hydra Lab users, please describe the impact and migration path.*

- [ ] Yes
- [ ] No

## Pull Request Description

**A few words to explain your changes**:  

### How you tested it
*Please make sure the change is tested, you can test it by adding UTs, do local test and share the screenshots, etc.*

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Technical design
- [ ] Build related changes
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Code style update (formatting, renaming) or Documentation content changes
- [ ] Other (please describe): 

### Feature UI screenshots or Technical design diagrams
*If this is a relatively large or complex change, kick it off by drawing the tech design with PlantUML and explaining why you chose the solution you did and what alternatives you considered, etc...*

### CODE REVIEW

File: react/webpack.config.js [hunk 1]
```diff
@@ -11,16 +11,16 @@
...
     target: devHydraLabServerEndpoint,
     secure: false,
     changeOrigin: true,
     // if you Hydra Lab Center Service enabled the OAuth, this node needed to be configured/uncommented.
     // The Authorization token is accessible in page Authentication -> TOKENS, click the add token button to create if there isn't one.
-    // headers: {
-    //     'Authorization': '*******',
-    // }
+    headers: {
+        'Authorization': '2a10ggIv7RB78EGCHlT58KFFCuGgBtPH52j5oOCzs77FqE6hnVpG1Zi',
```
[the remark this answers is not in the document]
Comment (DexterDreeeam) [reply react/webpack.config.js]: fixed, discard old token

File: react/src/css/common.css [hunk 2]
```diff
@@ -37,3 +37,8 @@
 .black .speaker				{background: linear-gradient(#555 0%, #777 80%)}
 .black .homeButtonBorder	{background: #888}
 .black .homeButton			{background: radial-gradient(#555 0%, #333 80%)}
+
+.agentDevices {
```
Comment (lifesaver0129) [inline react/src/css/common.css:41]: Where is agentDevices used?

File: react/src/component/DeviceDetailView.jsx [hunk 3]
```diff
@@ -158,7 +158,7 @@ export default class DeviceDetailView extends React.Component {
     }
 
     getApplePhoneCase(item) {
-        return <div className='deviceContainer' style={deviceStyle}>
+        return <div className='deviceDisplay' style={deviceStyle}>
```
Comment (lifesaver0129) [inline react/src/component/DeviceDetailView.jsx:161]: Confusing name of deviceDisplay, could we change it to deviceScreen or sth?

Comment (DexterDreeeam) [reply react/src/component/DeviceDetailView.jsx]: fixed

File: react/src/css/common.css (hunk 2 above)
Comment (DexterDreeeam) [reply react/src/css/common.css]: fixed

**Review by lifesaver0129 — APPROVED** [review]

File: react/src/component/DeviceAgentsView.jsx [hunk 4]
```diff
@@ -33,132 +35,114 @@ export default class DeviceAgentsView extends BaseView {
...
+            <div class='deviceAgents'>
+                <div class='deviceAgents-header'>
+                    <Typography variant="h4" className="mt-2 mb-2">Connected Devices</Typography>
+                    <div style={{ display: 'flex', gap: '16px' }}>
+                        <Button
+                            variant="outlined"
+                            className="pl-4 pr-4"
+                            endIcon={<span className="material-icons-outlined">auto_awesome_motion</span>}
+                            onClick={this.setAllCollapseStatus}>
+                            Collapse
```
Comment (hydraxman) [inline react/src/component/DeviceAgentsView.jsx:116]: Collapse All

Comment (DexterDreeeam) [reply react/src/component/DeviceAgentsView.jsx:116]: will fix it before next release

**Pull request merged**